.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: On Air Device CI
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
Just curious why are we leaving this blank?

Reply:
I haven't experimented with this extensively, but leaving this blank should allow this workflow to trigger on any pull request, instead of just the ones that we've declared (which is nothing)
Looking at the one above, the workflow will only trigger for `main` on a push to `main`, basically when we merge something into main!